Jenkinsfile

```diff
@@ -110,12 +147,6 @@ pipeline {
                         ARCH=`uname -m`;
                         printf "\n\n${MAGENTA}Platform: $ARCH ${RESET}\n"
 
-                        # Install nvm to easily set version of node to use
```

Comment:
@rajivnathan This cannot be removed as this is scoped for each stage. The unit test passed but the portal test failed because npm was not found https://ci.eclipse.org/codewind/job/Codewind/job/codewind/job/PR-554/22/console

Reply:
Yep, came to the same conclusion